app/models/landing_page_version.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: landing_page_versions
+#
+#  id           :integer          not null, primary key
+#  community_id :integer          not null
+#  version      :integer          not null
+#  released     :datetime
+#  content      :text(16777215)   not null
+#  created_at   :datetime
+#  updated_at   :datetime
+#
+# Indexes
+#
+#  index_landing_page_versions_on_community_id_and_version  (community_id,version) UNIQUE
+#
+
+class LandingPageVersion < ActiveRecord::Base
+end
```

Comment:
Both of these models are already defined in CustomLandingPage::LandingPageStore module. I'm fine with doing them with own model files (although it does feel redundant) but they should only be defined in one place.